Resources/Prototypes/_Goobstation/Objects/Misc/translator_implanters.yml

```
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+# SPDX-FileCopyrightText: 2025 ImWeax <59857479+ImWeax@users.noreply.github.com>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Gives ability to understand SpaceItalian. - Putting an implanter in for this would be bloat and I hate that I have to do any of this as an implant in the first place but doing it any other way would be a tremendous pain in the ass. I hate it here.
+#- type: entity
+#  id: BasicSpaceItalianTranslatorImplanter
+#  parent: BaseTranslatorImplanter
+#  name: basic space italian translator implanter
+#  components:
+#  - type: Implanter
+#    implant: BasicSpaceItalianTranslatorImplant
```

You should uncomment this and remove the unnecessary comment above it. It's not bloat if it serves a purpose, and, in this case, it serves the purpose of being an admin spawn implanter. If you don't want the implanters, for whatever reason, you don't have to add them since the implant itself is the only necessary part.
```suggestion
- type: entity
  id: BasicSpaceItalianTranslatorImplanter
  parent: BaseTranslatorImplanter
  name: basic space italian translator implanter
  components:
  - type: Implanter
    implant: BasicSpaceItalianTranslatorImplant
```